cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -130,7 +131,16 @@ func (s *Server) GetAvailablePackages(ctx context.Context, request *corev1.GetAv
 		}
 
 		log.Infof("Found repository: [%s], index URL: [%s]", name, url)
-		repoPackages, err := readPackagesFromRepoIndex(url)
+		repoRef := corev1.AvailablePackage_PackageRepositoryReference{
+			Name: name,
+		}
+		// namespace is optional according to https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/
```

Comment:
Namespace is optional when creating an object (as per the link), since you might do `kubectl -n foobar -f my-object.yaml` or you can specify the namespace in your object and apply it without the `-n foobar`, but in the case of the code here, we're getting an existing object. If the object is cluster-scoped, then it won't have a namespace, but otherwise I *think* it'd be very unexpected (ie. 500 internal) if we received a namespaced-scoped object without a namespace? Not 100%